src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala

```diff
@@ -265,18 +277,20 @@ object Boot extends IOApp with LazyLogging {
         googleIamDAO,
         googleStorageDAO,
         googleStorageNew,
-        googlePubSubDAO,
+        googleKeyCachePubSubDAO,
         config.googleServicesConfig,
         config.petServiceAccountConfig)
-    val notificationDAO = new PubSubNotificationDAO(googlePubSubDAO, config.googleServicesConfig.notificationTopic)
+    // Which pubSubProject/Dao should this be using?
+    val notificationDAO = new PubSubNotificationDAO(googleGroupSyncPubSubDAO, config.googleServicesConfig.notificationTopic)
```

Comment:
I wasn't sure which pubsubDAO this should use? Maybe we need a new project config value for that?